### ISSUE TITLE
Allow a user to open a tutorial alongside a sample

### ISSUE THREAD

**Issue by sstone1** [IBM-Blockchain/blockchain-vscode-extension#455]
The latest version of the extension allows you to import samples into the IDE, but there are no instructions for what to do with them (unless you go and find the relevant documentation from wherever).

It would be nice if a tutorial could be supplied with the sample (markdown?) that could be opened in an editor tab (on the right hand side?) that guides the user through using the sample and relevant IDE features (debug, test, etc).

Something like:
![screenshot 2019-01-17 at 14 58 50](https://user-images.githubusercontent.com/7118158/51328001-55cf1100-1a6a-11e9-9464-257be3cbfe2c.png)

**Comment by DennisM330** [comment]
Agreed. for example I just brought in the Fabcar sample. then installed the smart contract, and instantiated it calling the initLedger method. Next steps is I generated tests which is a very quick way to perform your initial testing of your Smart Contract. Then I modify the generated test module to display requests from when methods are invoked. For example, for queryCar  I added:  const resultJSON = JSON.parse(response.toString());
  console.log(args + ' details: ' + resultJSON);
for queryAllCars, I added: const resultJSON = JSON.parse(response.toString());
        console.log('All Cars List: ' + resultJSON);
